torch/csrc/jit/serialization/import.cpp

```diff
@@ -83,10 +83,34 @@ IValue readArchiveAndTensors(
     return len;
   };
 
-  std::string archive_name_plus_slash = archive_name + "/";
+  const std::string slash = "/";
+  std::string archive_name_plus_slash = archive_name + slash;
   auto read_record = [&](const std::string& name) {
-    std::string ss = archive_name_plus_slash + name;
-    return std::get<0>(stream_reader.getRecord(ss));
+    std::size_t found = name.find(slash);
```

Comment:
Why does this need to be changed? This is full JIT import and we have not touched this right?

Reply:
Good point! I thought we also use this such that `torch.jit.load` can import `model.ptl`. I might be wrong though. Let me double check.

Reply:
Yes we can load a ptl model vis torch.jit.load but that path will not read bytecode.